css/main.css

```diff
@@ -423,6 +424,23 @@ button {
     color: #3B81F5;
 }
 
+.past-speakers-link {
+    position: absolute;
+    bottom: 0;
+    right: 20px;
+    font-size: .7em;
+}
+
+.past-event {
+    list-style-type: none;
+    position: relative;
+    padding-left: 20px;
+}
+
+.past-event .event-speaker p {
+    color: #779;;
```

Comment:
Thanks for the sharp eyes, Paul! I'll remove that and add the rest of the past speakers soon. :)